### ISSUE TITLE
GRAVITY RUSH

### ISSUE THREAD

**Issue by Zangetsu38** [Vita3K/compatibility#20]
# Game summary
- Game name: GRAVITY RUSH
- Game serial: PCSF00024

# Vita3K summary
- Version: v0.1
- Build number: 242-ddb3595
- Commit hash: https://github.com/vita3k/vita3k/commit/e5ca809 

# Test environment summary
- Tested by: Zangetsu38
- OS: Windows 10 17.134.5
- CPU: AMD Ryzen 7 2700X
- GPU: Nvidia MSI GTX 1080 Ti Trio 
- RAM: 16 GB DDR4

# Issues
Crash after start boot with rose screen

# Screenshots
Not for moment

# Log
[Mai Decrypted vita3k.log](https://github.com/Vita3K/compatibility/files/1972860/vita3k.log)

[Mai Dump vita3k.log](https://github.com/Vita3K/compatibility/files/1972861/vita3k.log)

# Recommended labels
Crash

**Comment by Zangetsu38** [comment]
# Vita3K summary
- Version: v0.1
- Build number: 1071
- Commit hash: https://github.com/vita3k/vita3k/commit/205c3ac

# Test environment summary
- Tested by: Zangetsu38
- OS: Windows 10
- CPU: AMD Ryzen 2700X
- GPU: NVIDIA GTX 1080 Ti
- RAM: 16 GB DDR4 3600 mhz

# Issues
- stop boot on unicron error :
>Unicorn error 0x7 at: start PC: 0x81525461 error PC 0x8127D448 LR: 0x8127E9C7

# Log
[vita3k.log](https://github.com/Vita3K/compatibility/files/3266677/vita3k.log)

**Comment by DIO-Brando-explains-the-joke** [comment]
is this project going to end?

**Comment by ZuwasieMan** [comment]
Vita3K summary

    Version: v0.1.5 2632-37241b1f

Test environment summary

    OS: Windows 10 Pro
    CPU: AMD Ryzen 9 5950x
    GPU: AMD 6900 XT
    RAM: 32 GB DDR4

Test Result

Intro plays very good. After that goes In-Game, only a couple of the environment textures are being displayed, but the character itself is playable (that means, I can move the character in the environment). Framerate seems very good as well (28-30 fps) with ocassional stuttering due to shader compilation. Sound seems to work fine. 2d artwork (dialogue between characters) does not render correctly.

**Comment by Zangetsu38** [comment]
# Vita3K summary
- Version: v0.1.8
- Build number: 3191
- Commit hash: https://github.com/vita3k/vita3k/commit/a197d981
- CPU backend: Dynarmic
- GPU backend: Vulkan

# Issue
- Some flish light  in element in certain distance
- gyroscope feature mussing (motion)
- crash in same plan all time in debut of episode 2

# Screenshot
![image](https://github.com/Vita3K/compatibility/assets/5261759/0cb4c41d-41f2-4202-b48a-90b76ae21f4c)
![image](https://github.com/Vita3K/compatibility/assets/5261759/bd31f998-a9f6-4aeb-afb8-25912fd46690)
![image](https://github.com/Vita3K/compatibility/assets/5261759/b5b7b909-a9c4-4394-8312-b57c941331af)
![image](https://github.com/Vita3K/compatibility/assets/5261759/117a3dab-fc45-4625-82ad-3aecf77b5741)
![image](https://github.com/Vita3K/compatibility/assets/5261759/83a5e7d8-8730-4246-82b2-1acba4db6f52)
![image](https://github.com/Vita3K/compatibility/assets/5261759/7aba3521-167d-4e4a-8614-7ec8995c5212)

# Log
[vita3k.log](https://github.com/Vita3K/compatibility/files/11583535/vita3k.log)

**Comment by ArbestRiAgain** [comment]
From my testing I was able to get to Episode 3

**Comment by Zangetsu38** [comment]
> From my testing I was able to get to Episode 3

yes, after testn crash is caused all time is same time by surface sync option enable

**Comment by ThomasHineXYZ** [comment]
Has there been any improvement for this game?

**Comment by purevg** [comment]
Gravity rush worked very well until we go through "Rift Plane" in "Episode 6: The lost city". The end of the episode 5 also was very in high brightness with contours very hard to distinguish.

![image](https://github.com/Vita3K/compatibility/assets/95699360/3d7b9d5c-17c9-4177-98be-ab262aa742f8)

Also the we are unable to activate the gravity slide with a dual shock 4 gamepad, is there any way to workaround this ?

Thank you !

==============
OS: Debian GNU/Linux 12 (bookworm)
Version: Vita3K v0.2.0 3608-53741632
CPU: AMD Ryzen 5 5600 6-Core Processor | 12 Threads | 4.225 GHz
Vulkan device: AMD Radeon RX 6600 (RADV NAVI23)

**Comment by Slade1972** [comment]
# Vita3K summary
- Version: v0.2.0
- Build number: 3691
- Commit hash: https://github.com/vita3k/vita3k/commit/31826697
- CPU backend: Dynarmic
- GPU backend: Vulkan

# Test environment summary
- Tested by: Slade 
- OS: Microsoft Windows 10 Professional (build 19045)
- CPU: Intel(R) Core(TM) i7-3770K CPU @ 3.50GHz
- GPU: NVIDIA GeForce GTX 1060 6GB
- RAM: 16 GB

# Issues
- Colours / brightness in some sections of the game are incorrect or over exposed.
- There's a whole section just after you meet the fortune teller where the entire level is almost completely white.
- Shadows and bright spots on the ground flicker in and out depending on character and camera movement.
- Motion controls work on a Dualsense controller.


# Screenshots

![gr1](https://github.com/user-attachments/assets/33426aa1-d07d-4170-a9de-30b33db5d8ae)
![gr2](https://github.com/user-attachments/assets/a875cdea-f6fe-4373-8f33-b435a3a18170)
![gr3](https://github.com/user-attachments/assets/0695e341-6db8-42b4-b942-66e34f2f8f22)
![gr4](https://github.com/user-attachments/assets/0177dcb6-043b-40be-a689-8e28d4844eaf)
![gr5](https://github.com/user-attachments/assets/62adf99a-6e40-488a-af0f-a439cded4511)


# Log
[PCSF00024 - [GRAVITY RUSH™].log](https://github.com/user-attachments/files/18220798/PCSF00024.-.GRAVITY.RUSH.log)


# Recommended labels
- Ingame +
- graphics bug
